### PR TITLE
swww: add page

### PR DESCRIPTION
**Pull request by rexept**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**
0.9.5

### CODE REVIEW

File: pages/linux/swww.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+> Efficient animated wallpaper daemon for wayland.
+> See also: `swww-daemon`.
+> More information: <https://github.com/LGFae/swww>.
+
+- Set wallpaper:
+
+`swww img {{/path/to/image}}`
+
+- Set wallpaper to specified [o]utputs:
```
Comment (debghs) [inline pages/linux/swww.md:11]:
```suggestion
- Set wallpaper to specified outputs:
```
shouldn't it be like this?

Comment (spageektti) [reply pages/linux/swww.md:11]: No, see style guide.

File: pages/linux/swww.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# swww
+
+> Efficient animated wallpaper daemon for wayland.
+> See also: `swww-daemon`.
+> More information: <https://github.com/LGFae/swww>.
+
+- Set wallpaper:
+
+`swww img {{/path/to/image}}`
```
Comment (spageektti) [inline pages/linux/swww.md:9]:
```suggestion
`swww img {{path/to/image}}`
```
Can it be relative?

**Review by spageektti — COMMENTED** [review]
Hi. Thanks for your contribution.

File: pages/linux/swww.md (hunk 1 above)
Comment (debghs) [reply pages/linux/swww.md:11]: Ahh, I see. No worries, then.

File: pages/linux/swww.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+> See also: `swww-daemon`.
+> More information: <https://github.com/LGFae/swww>.
+
+- Set wallpaper:
+
+`swww img {{path/to/image}}`
+
+- Set wallpaper to specified [o]utputs:
+
+`swww img -o {{output1,output2,...}} {{/path/to/image}}`
```
Comment (spageektti) [inline pages/linux/swww.md:13]:
```suggestion
`swww img -o {{output1,output2,...}} {{path/to/image}}`
```

**Review by debghs — APPROVED** [review]
LGTM.

**Review by spageektti — APPROVED** [review]

File: pages/linux/swww.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# swww
+
+> Efficient animated wallpaper daemon for wayland.
```
Comment (spageektti) [inline pages/linux/swww.md:3]:
```suggestion
> Efficient animated wallpaper daemon for Wayland.
```

**Review by spageektti — APPROVED** [review]

**Pull request merged**